torch_spyre/_inductor/codegen/compute_ops.py

```diff
@@ -84,27 +103,49 @@ def generate_sfp_op(pointers, *, op, dimensions, inputs, outputs, reduction, **k
                 {
                     op: {
                         "numCoresUsed_": cores,
-                        "numCoreletsUsed_": 1,
+                        "numCoreletsUsed_": corelets,
                         "coreIdsUsed_": [i for i in range(cores)],
                         "N_": {
                             "name_": "n",
                             "mb_": dimensions[0] if d2 else 0,
                             "x_": dimensions[1] if d3 else 0,
                             "out_": dimensions[-1],
                         },
-                        "dataStageParam_": {
+                        "dataStageParam_": {  # TODO: extend coreletSplit to support splitting on "mb" cases
                             "0": {
                                 "ss_": {
                                     "name_": "core",
                                     "mb_": dimensions[0] if d2 else 0,
                                     "x_": dimensions[1] if d3 else 0,
                                     "out_": dimensions[-1] // cores,
+                                    "coreletSplit_": {
+                                        "out": [
```

Comment:
don't we have the dim order already somewhere? can we use the first dimension in that array as the one where the split is happening?